### ISSUE TITLE
mitophagy in N starvation (was FYPO:0007448)

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#3876]
I just used

atg43 | atg43-1(atg43::kanMX4(nt49-203)) | FYPO:0007448 | normal reticulophagy during nitrogen starvation | Western blot assay | nitrogen absent | fig1

but I think this can  change to 

FYPO:0007448 | normal reticulophagy

since N starvation is just the condition used to induce autophagy generally.
and  the corresponding mitophagy terms used in the same session are

mim2 | mim2Δ | FYPO:0000500 | decreased mitophagy | Western blot assay | nitrogen absent |  

(FYPO:0007448 had not yet been used - not sure why, I will check if I forgot to add to the session requested for, but there is no annotation to check for conditions)

**Comment by ValWood** [comment]
I couldn't find the original request for "normal reticulophagy during nitrogen starvation", so not sure why it wasn't used yet.

**Comment by mah11** [comment]
> I think this can change to
> 
> FYPO:0007448 | normal reticulophagy
> 
> since N starvation is just the condition used to induce autophagy generally.

Does that mean it would also be OK to move it to the "vegetative growth" department?

> I couldn't find the original request for "normal reticulophagy during nitrogen starvation",

oops, I forgot to paste the IDs in when I did it - it's #3777

**Comment by ValWood** [comment]
Hmm, I'd better check that. These cells are in vegetative growth, but I wonder why they don't enter quiescence if they are nitrogen starved.....

**Comment by ValWood** [comment]
Scub this they do enter quiescence. The ones which aren't are nitrogen is readded, they return to vegetative growth but still lose viability.

So I guess for consistency we should have 
"normal  mitophagy during nitrogen starvation"
"abolished mitophagy during nitrogen starvation"
"decreased mitophagy during nitrogen starvation"
"increased mitophagy during nitrogen starvation"

**Comment by mah11** [comment]
good thing you clocked that before I made changes! ok, will add new mitophagy terms

**Comment by ValWood** [comment]
I still don't fully understand what stage the cells are in, even after looking at some previous papers, but better to keep the detail...

**Comment by mah11** [comment]
OK!

**Comment by mah11** [comment]
I've added the N starvation mitophagy terms (and one grouping term), but haven't made any other changes to any *phagy terms, including FYPO:0007448. (E.g. I haven't added any "vegetative growth" terms for autophagy or mitophagy, but I have left the abnormal reticulophagy terms under vegetative growth.) I think that's consistent with what's here and in the emails. If any further changes become necessary, let's put them in a new ticket.

I saved the emails into a file that I can attach here if you think it'll be useful for future reference (and OK to put on a ticket that's publicly accessible).

new terms:
  abnormal mitophagy FYPO:0007590
  normal mitophagy FYPO:0007591
  normal mitophagy during nitrogen starvation FYPO:0007592
  abnormal mitophagy during nitrogen starvation FYPO:0007593
  abolished mitophagy during nitrogen starvation FYPO:0007594
  increased mitophagy during nitrogen starvation FYPO:0007595
  decreased mitophagy during nitrogen starvation FYPO:0007596
